True OOTB but we work around this by using __attribute__((sysv_abi)) in 
libgcrypt-use-correct-asm-code-for-win64.patch

--- origsrc/libgcrypt-1.6.6/mpi/config.links	2016-03-29 06:54:58.000000000 -0500
+++ src/libgcrypt-1.6.6/mpi/config.links	2016-09-23 14:58:15.337558800 -0500
@@ -134,11 +134,6 @@ case "${host}" in
 	path="amd64"
         mpi_cpu_arch="x86"
 	;;
-    x86_64-*mingw32*)
-	echo '/* No working assembler modules available */' >>./mpi/asm-syntax.h
-	path=""
-	mpi_cpu_arch="x86"
-        ;;
     x86_64-*-*)
 	echo '#define ELF_SYNTAX' >>./mpi/asm-syntax.h
 	cat  $srcdir/mpi/i386/syntax.h	    >>./mpi/asm-syntax.h
